src/API/socket/device-events.js

```diff
@@ -74,6 +77,10 @@ export default function(io){
             debug('Socket disconnected');
         }
 
-    }
+        function isAllowedEventToSend(event) {
+            let allowedEvents = ['status', 'device-add'];
 
+            return allowedEvents.indexOf(event) !== -1;
```

Comment:
Actually, I thought about it, but unfortunately, .includes() currently isn't part of specification ES6.
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/includes